src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
File comment

Reply:
Updated in c7edce73cbc8f57b149a0692c5b2d738fcbda5d5